internal/usecase/daemon/copy_scroll_animation_test.go

```
@@ -0,0 +1,132 @@
+			clock := newCoordinatorMockClock(t, 16)
+			f.d.clock = clock.clock
+			emitted := make(chan struct{}, 16)
+			f.ac.renderStages.emit = func() { emitted <- struct{}{} }
+			await := func() {
+				select {
+				case <-emitted:
+				case <-time.After(time.Second):
+					t.Fatal("scroll frame not emitted")
+				}
+				f.ac.sendMu.Lock()
+				f.ac.sendMu.Unlock()
```

golangci-lint (2.13.2)

[error] SA2001: empty critical section

(staticcheck)